<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Caesar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Vigenere</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>AES</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>701</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>701</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>58</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Plaintext</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>207</y>
      <width>58</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cypher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>DES</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RSA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Encryption</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Decription</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>XOR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Base64</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>MD5</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Something to add</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Caesar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Vigenere</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>AES</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DES</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RSA</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>XOR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Base64</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MD5</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>542</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Gernerate RSA key</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>526</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
